targets/LuaSdk/SharedTesting/PlayFabTesting/PlayFabApiTestSuite.lua

```diff
@@ -348,7 +349,10 @@ function PlayFabApiTestSuite.OnGetEntityToken(result)
     if (result.Entity) then 
         PlayFabApiTestSuite.entityId = result.Entity.Id
         PlayFabApiTestSuite.entityType = result.Entity.Type
-        AsyncTestSuite.EndTest("PASSED", result.Entity.Id)
+        if (result.EntityToken) then
+            PlayFabSettings._internalSettings.EntityToken = result.EntityToken.EntityToken
```

Comment:
This is probably a case where Lua SDK is so old that it's missing code that has been updated in all other SDKs.
Please remove this EntityToken assignment here, and apply this assignment somewhere appropriate in the PlayFabApi.lua.ejs file. Doing it here means it's only set once for this test, and not properly set "in general" which really defeats the entire point of this test.